tdms/src/iterator.cpp

```diff
@@ -6269,60 +6227,6 @@ void extractPhasorHNorm(complex<double> *Hnorm, double ft, int n, double omega,
   *Hnorm += ft * exp(fmod(omega * ((double) n + 0.5) * dt, 2 * dcpi) * I) * 1. / ((double) Nt);
 }
 
-//these indices are set according to the electric part of the pml - since the index of the
-//first cell is different in the case of the electric update eqns.
-
-//i_l is the index into the fdtd grid which is the first non-pml cell in the i direction
-//i_u is the index into the fdtd grid which is the last non-pml cell in the i direction
-//
-//result gives field according to the exp(-iwt) convention
-void extractPhasorsVolumeH(MagneticField &H, MagneticSplitField &H_s,
-                           int n, double omega, double dt, int Nt) {
-
-  complex<double> phaseTerm, subResult;
-  double hx_m, hy_m, hz_m;
-
-  //a + 0.5 is added because we always know H half a time step
-  //after we know E.

```

Comment:
And maybe add this to the [ElectricField::phase](https://github.com/t-young31/ifdtd/pull/7/commits/ce7b7fdec9a8a4abf476db7ec7b15d1f410100e7#diff-07de152f1fee2c5f2062108ebe267fb502b782270603d5d02037f8fba56f57dcR7) ?

(I don't insist)